packages/devtools_app/lib/src/screens/inspector_v2/widget_properties/properties_view.dart

```diff
@@ -80,27 +75,6 @@ class PropertyItem extends StatelessWidget {
     );
   }
 
-  BorderRadius _calculateBorderRadiusForRow({
-    required int rowIndex,
-    required int totalRows,
-  }) {
-    // Rounded top corners for the first row:
-    if (rowIndex == 0) {
-      return const BorderRadius.only(
-        topLeft: defaultRadius,
-        topRight: defaultRadius,
-      );
-    }
-    // Rounded bottom corners for the last row:
-    if (rowIndex == (totalRows - 1)) {
-      return const BorderRadius.only(
-        bottomLeft: defaultRadius,
-        bottomRight: defaultRadius,
-      );
-    }
-    return BorderRadius.zero;
-  }
-
   Border _calculateBorderForRow({
```

Comment:
if we remove the right and left borders here, and then wrap the whole list in a OutlineDecoration widget, this should give us the look we are looking for where the entire list has a border and the list items are separated by a border

Reply:
Switched to using a tabbed view here (see comment above)